middleware/common/include/common/buffer/type.h

```diff
@@ -184,13 +188,15 @@ namespace casual
          inline buffer::handle::type handle() const noexcept { return payload.handle();}
          inline buffer::handle::mutate::type handle() noexcept { return payload.handle();}
 
-         //inline friend bool operator == ( const Buffer& lhs, buffer::handle::type rhs) { return lhs.handle() == rhs;}
+         bool inbound() const { return m_inbound;}
```

Comment:
could be `noexcept`

Reply:
Yeah, but I don't think noexcept improve that much. Besides move semantics and such. I also believe that the compiler can deduce inline getters as conceptual noexcept. With noexcept the compiler has to add guards against exceptions.
I'm not so sure about sprinkle noexcept all over any more...